s30566-DanielMakowski/zajecia3/zadanie_domowe2.txt

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#Przygotuj wyrażenie regularne sprawdzające czy tekst jest kodem pocztowym (cały tekst, czyli zaczyna się od cyfry i kończy się cyfrą).
+
+postal=$1
+
+if [[ $postal =~ ^[0-9]{2}[\-][0-9]{3}$ ]]; then
```

Comment:
Jeśli masz obowiązek wstawić gdzieś jakiś konkretny znak to nie rób z tego słownika jednoznakowego. To nie jest konieczne.